docs/reST/tut/newbieguide.rst

```diff
@@ -327,7 +329,7 @@ still be sitting in the event queue, however, waiting to be retrieved and
 processed.
 
 The lesson is: choose the system that meets your requirements.  If you don't
-have much going on in your loop -- say you're just sitting in a ``while True``
+have much going on in your loop -- say you're just sitting in a ``while 1``
```

Comment:
Why is this change being made? IMHO `while True` seems more pythonic

Reply:
I agree, most of this is just copied from the original PR.